inc/template-functions.php

```diff
@@ -75,7 +75,7 @@ function twentynineteen_get_the_archive_title() {
 	} elseif ( is_tax() ) {
 		$tax = get_taxonomy( get_queried_object()->taxonomy );
 		/* translators: 1: Taxonomy singular name */
-		$title = sprintf( __( '%s Archives: ' ), $tax->labels->singular_name );
+		$title = sprintf( __( '%s Archives: ', 'twentynineteen' ), $tax->labels->singular_name );
```

Comment:
Does this need a space after `:`? None of the others have it.

Reply:
Not sure. :) It might need